src/controllers/user/createUser.ts

```diff
@@ -69,7 +81,7 @@ export const createUser = async (req: Request, res: Response) => {
 
   } catch (err) {
     console.log("Error creating user:", err.message)
```

Comment:
remove console logs

Reply:
Lots of files prolly have this. Will remove them.

Reply:
I think it would be a better idea to just comment them out. Might help in future.